src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/ManifestTest.cs

```diff
@@ -964,5 +966,60 @@ class TestService : Service {{ public override Android.OS.IBinder OnBind (Androi
 			Assert.IsTrue (b.LastBuildOutput.ContainsText ($"AndroidManifest.xml(12,5): java{extension} error AMM0000:"), "Should recieve AMM0000 error");
 			Assert.IsTrue (b.LastBuildOutput.ContainsText ("Apps targeting Android 12 and higher are required to specify an explicit value for `android:exported`"), "Should recieve AMM0000 error");
 		}
+
+		static object [] SupportedOSTestSources = new object [] {
+			new object[] { "", true },
```

Comment:
Please use the convention used earlier in this file, which includes a "comment" for each paramerter:

```csharp
new object[]{
    /* minSdkVersion */        "",
    /* removeUsesSdkElement */ false,
},
new object[]{
    /* minSdkVersion */        "21.0",
    /* removeusesSdkElement */ true,
},
// …
```